src/runner/general_settings.cpp

```diff
@@ -11,6 +11,13 @@
 static std::wstring settings_theme = L"system";
 static bool run_as_elevated = false;
 
+// TODO: add resource.rc for settings project and localize
+namespace localized_strings
+{
+    const std::wstring_view STARTUP_DISABLED_BY_POLICY = L"Disabled by admin policy.";
+    const std::wstring_view STARTUP_DISABLED_BY_USER = L"Disabled by user.";
```

Comment:
I think we should improve both messages. In particular for the second message we may want to add a reference to the TaskManager/Windows Settings (if we make it work we will also add the direct link to the Windows Settings but currently it seems WebView doesn't like it).

@crutkas suggestions?

Reply:
can i see a screenshot how this is displayed to the end user?

Reply:
![image](https://user-images.githubusercontent.com/3206696/73484949-839ba500-43a2-11ea-8b15-d59363fd7f0c.png)

Reply:
how does this sound

**by group policy:**
"This setting has been disabled by your administrator"

**by user:**
"Please re-enable via \[Startups settings\](LINK)" <- not thrilled with this
"This setting has been disabled via \[Startups settings\](LINK), please re-enable there for this to work" <- too wordy IMO

Change the look/feel to match what defender does
![image](https://user-images.githubusercontent.com/1462282/73500710-47ebe580-4378-11ea-8505-9f18da7c73f7.png)

Reply:
> "Please re-enable via [Startups settings](LINK)"

Not thrilled as well, seems like an order ;)

> "This setting has been disabled via [Startups settings](LINK), please re-enable there for this to work"

What about using only the first part "This setting has been disabled via [Startups settings](LINK)" if we assume the user has done it, it's also true they know they did it and they are not confused by the fact we redirect them there, without having to explain to go there.

Reply:
![image](https://user-images.githubusercontent.com/1828123/73546763-d03cba00-444e-11ea-8544-a320fdca2d22.png)

Reply:
Looks good, but before resolving the conversation, let's wait for final review by @crutkas

Reply:
i like that, great collaboration here